getting-started/bookshelf/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/GoogleCloudPlatform/golang-samples/getting-started/bookshelf
+
+go 1.13
+
+require (
+	cloud.google.com/go v0.44.3
+	github.com/gofrs/uuid v3.2.0+incompatible
```

Comment:
Perhaps consider switching to another version that isn't incompatible.

For example, `github.com/gofrs/uuid/v3 v3.1.2` or `github.com/google/uuid v1.1.1`.

Reply:
Unfortunately, that version also doesn't work. So, sticking with incompatible.